test/induction/inductive-implicits-bench.scala

```diff
@@ -0,0 +1,619 @@
+    Int ::
+    Int ::
+//
+    Int ::
+    Int ::
+    Int ::
+    Int ::
+    Int ::
+    Int ::
+    Int ::
+    Int ::
+    Int ::
+    Int ::
+//
+    Int ::
+    Int ::
+    Int ::
+    Int ::
+    Int ::
+    Int ::
+    Int ::
+    Int ::
+    Int ::
+    Int ::
+//
+    Int ::
+    Int ::
+    Int ::
+    Int ::
+    Int ::
+    Int ::
+    Int ::
+    Int ::
+    Int ::
+    Int ::
+//
+    Int ::
+    Int ::
+    Int ::
+    Int ::
+    Int ::
+    Int ::
+    Int ::
+    Int ::
+    Int ::
+    Int ::
+//
+    Int ::
+    Int ::
+    Int ::
+    Int ::
+    Int ::
+    Int ::
+    Int ::
+    Int ::
+    Int ::
+    Int ::
+//
+    Int ::
+    Int ::
+    Int ::
+    Int ::
+    Int ::
+    Int ::
+    Int ::
+    Int ::
+    Int ::
+    Int ::
+//
+    Int ::
+    Int ::
+    Int ::
+    Int ::
+    Int ::
+    Int ::
+    Int ::
+    Int ::
+    Int ::
+    Int ::
+//
+    Int ::
+    Int ::
+    Int ::
+    Int ::
+    Int ::
+    Int ::
+    Int ::
+    Int ::
+    Int ::
+    Int ::
+//
+    Int ::
+    Int ::
+    Int ::
+    Int ::
+    Int ::
+    Int ::
+    Int ::
+    Int ::
+    Int ::
+    Int ::
+//
+    Int ::
+    Int ::
+    Int ::
+    Int ::
+    Int ::
+    Int ::
+    Int ::
+    Int ::
+    Int ::
+    Int ::
+//
+    Int ::
+    Int ::
+    Int ::
+    Int ::
+    Int ::
+    Int ::
+    Int ::
+    Int ::
+    Int ::
+    Int ::
+//
+    Int ::
+    Int ::
+    Int ::
+    Int ::
+    Int ::
+    Int ::
+    Int ::
+    Int ::
+    Int ::
+    Int ::
+//
+    Int ::
+    Int ::
+    Int ::
+    Int ::
+    Int ::
+    Int ::
+    Int ::
+    Int ::
+    Int ::
+    Int ::
+//
+    Int ::
+    Int ::
+    Int ::
+    Int ::
+    Int ::
+    Int ::
+    Int ::
+    Int ::
+    Int ::
+    Int ::
+//
+    Int ::
+    Int ::
+    Int ::
+    Int ::
+    Int ::
+    Int ::
+    Int ::
+    Int ::
+    Int ::
+    Int ::
+//
+    Int ::
+    Int ::
+    Int ::
+    Int ::
+    Int ::
+    Int ::
+    Int ::
+    Int ::
+    Int ::
+    Int ::
+//
+    Int ::
+    Int ::
+    Int ::
+    Int ::
+    Int ::
+    Int ::
+    Int ::
+    Int ::
+    Int ::
+    Int ::
+//
+    Int ::
+    Int ::
+    Int ::
+    Int ::
+    Int ::
+    Int ::
+    Int ::
+    Int ::
+    Int ::
+    Int ::
+//
+    Int ::
+    Int ::
+    Int ::
+    Int ::
+    Int ::
+    Int ::
+    Int ::
+    Int ::
+    Int ::
+    Int ::
+//
+    Int ::
+    Int ::
+    Int ::
+    Int ::
+    Int ::
+    Int ::
+    Int ::
+    Int ::
+    Int ::
+    Int ::
+//
+    Int ::
+    Int ::
+    Int ::
+    Int ::
+    Int ::
+    Int ::
+    Int ::
+    Int ::
+    Int ::
+    Int ::
+//
+    Int ::
+    Int ::
+    Int ::
+    Int ::
+    Int ::
+    Int ::
+    Int ::
+    Int ::
+    Int ::
+    Int ::
+//
+    Int ::
+    Int ::
+    Int ::
+    Int ::
+    Int ::
+    Int ::
+    Int ::
+    Int ::
+    Int ::
+    Int ::
+//
+    Int ::
+    Int ::
+    Int ::
+    Int ::
+    Int ::
+    Int ::
+    Int ::
+    Int ::
+    Int ::
+    Int ::
```

Comment:
you could write these 10 per line..
easier to read, easier to count, lose the every-10-lines separator.. win win win

Reply:
Fair enough, but `55j` works pretty well in vim ;-)